### ISSUE TITLE
Files icons are visible when they're disabled

### ISSUE THREAD

**Issue by lamartire** [kyazdani42/nvim-tree.lua#680]
Hello! I've faced with a problem after the latest version install – icons are still visible even they're hidden.

There is the plugin configuration:

```lua
vim.g.nvim_tree_show_icons = {
  folders = 0,
  files = 0,
  git = 0,
  folder_arrows = 0,
}

require("nvim-tree").setup({
  update_focused_file = {
    enable = 1,
  },
  view = {
    width = 50,
    auto_resize = 1,
  },
})
```

How they're actually looking:

![image](https://user-images.githubusercontent.com/13414205/135728771-f78c0edc-402c-4db6-852d-d129c6d7ba52.png)

**Comment by gegoune** [comment]
Duplicate #669

**Comment by kyazdani42** [comment]
nope this is actually a bug just fixed it in `1.4.17` (4de55440b8). Not logical to show a default icon by default when you don't want to show default icon :)
